spec/rails_helper.rb

```diff
@@ -66,174 +66,85 @@
+            sharePhone
+            shareEmail
+            aidRequests {
+              id
+              organizationId
+              aidType
+              language
+              description
+              status
+              organization {
+                name
+              }
+            }
+          }
+        }
+      GRAPHQL
+    end
+  
+    def get_organizations_by_city_state
+      <<-GRAPHQL
+        query getAllOrgs($city: String!, $state: String!) {
+          organizations(city: $city, state: $state) {
+            id
+            name
+            contactPhone
+            contactEmail
+            streetAddress
+            website
+            city
+            state
+            zip
+            latitude
+            longitude
+            shareAddress
+            sharePhone
+            shareEmail
+            aidRequests {
+              id
+              organizationId
+              aidType
+              language
+              description
+              status
+              organization {
+                name
+              }
+            }
+          }
+        }
+      GRAPHQL
+    end
 
-    @refugee_shelter_names = [
-      "Safe Haven Refuge",
-      "Hopeful Haven Shelter",
-      "Harbor of Safety",
-      "Rescue Refuge Center",
-      "New Beginnings Shelter",
-      "Warmth and Welcome Shelter",
-      "Pathway to Peace Refuge",
-      "Compassion Corner",
-      "Emerald Oasis Shelter",
-      "Healing Hearts Haven",
-      "Solace Shelter",
-      "Bridge to Belonging",
-      "Sunrise Sanctuary",
-      "Unity Refuge Center",
-      "Caring Haven",
-      "Serenity Shelter",
-      "Community Compass Shelter",
-      "Harmony Haven",
-      "Sunny Days Refuge",
-      "Lighthouse of Hope",
-      "Angel's Arms Shelter",
-      "Life Renewal Refuge",
-      "Comfort Cove Shelter",
-      "Refugee Empowerment Center",
-      "Graceful Shelter",
-      "Tranquil Oasis",
-      "Dream Safe Haven",
-      "Shelter from the Storm",
-      "Open Arms Refuge",
-      "Heavenly Harbor",
-      "Empathy Haven",
-      "Journey to Belonging",
-      "Heartwarming Shelter",
-      "Sanctuary of Dreams",
-      "Beloved Shelter",
-      "Recovery Refuge",
-      "Hope Bridge Shelter",
-      "Peaceful Pathway Shelter",
-      "Warm Embrace Refuge",
-      "Safe Passage Shelter",
-      "Sheltered Serenity",
-      "Benevolent Refuge",
-      "Radiant Retreat Shelter",
-      "Hearts United Haven",
-      "Companionship Refuge",
-      "Golden Oasis Shelter",
-      "Welcome Winds Refuge",
-      "Guiding Light Shelter",
-      "Promise of Tomorrow Refuge",
-      "Homecoming Haven",
-      "Bridges of Hope Shelter",
-      "Whispering Pines Refuge",
-      "New Horizons Shelter",
-      "Strength in Unity Haven",
-      "Sheltered Hearts Sanctuary",
-      "Cherished Dreams Refuge"
-    ]
+    @org_1 = create(:organization)
+    3.times do
+      create(:aid_request, organization: @org_1)
```

Comment:
Instead of 3.times do-
You can write `create_list(:aid_request, 3, organization: @org_1)`